.github/ISSUE_TEMPLATE/release.md

```diff
@@ -74,12 +65,6 @@ Note: Do not publish draft releases from PR branches, because those branches wil
   - [ ] Subscan team. Ensure subscan service can continue to scan calamari blocks.
 
 ## Notes
-
-### Burn In
-
-- [ ] Ensure that Manta DevOps has run the new release on Baikal nodes for at least 3 days prior to publishing the release.
-- [ ] Check all testnet nodes [on Grafana](https://grafana.pulse.pelagos.systems/d/rYdddlPWk/node-exporter-full?orgId=1&refresh=1m&var-DS_PROMETHEUS=default&var-job=calamari-testnet%20invulnerable%20collator%20(ssl)&var-node=crunchy.baikal.testnet.calamari.systems:443&var-diskdevices=%5Ba-z%5D%2B%7Cnvme%5B0-9%5D%2Bn%5B0-9%5D%2B%7Cmmcblk%5B0-9%5D%2B) for irregularities since the upgrade
-
```

Comment:
Why is this removed, previous sections link to it.

Reply:
The explanations here are duplicated with the instructions above, i'll remove the link